### PR TITLE
Daily attendance frontend

### PR DESCRIPTION
**Pull request by Asna-A**

### CODE REVIEW

File: src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  getDailyAttendence()
+  {
+    //const employeeId=localStorage.getItem('employeeId');
+    const employeeId=1;
+    const selectedDate=new Date('2024-11-11')
+    const date=formatDate(selectedDate, 'yyyy-MM-dd', 'en-US');
+    this.service.getDailyAttendence(employeeId,date).subscribe(
+      (response: DailyAttendance) =>{
+        if(response){
+          console.log(response)
```
Comment (GSARYAPUTHRAN) [inline src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts:47]: remove

File: src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          this.dailyData=response;  
+      }
+        else {console.error('Error fetching attendance:') }
```
Comment (GSARYAPUTHRAN) [inline src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts:50]: dont use console

File: src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  convertToTime(dateStr: string): string {
+    const date = new Date(dateStr); 
+    const hours = date.getHours().toString().padStart(2, '0'); 
+    const minutes = date.getMinutes().toString().padStart(2, '0'); 
+    return `${hours}:${minutes}`; 
+  }
+  downloadDailyReport()
+  {
+    //const employeeId=localStorage.getItem('employeeId');
+    const employeeId=1;
```
Comment (GSARYAPUTHRAN) [inline src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts:71]: why?

**Review by GSARYAPUTHRAN — APPROVED** [review]

File: src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts (hunk 3 above)
Comment (Asna-A) [reply src/ui/src/app/dashboard/dashboard-body/main/attendence-module/daily-attendence/daily-attendence.component.ts:71]: the employeeId should be stored by the profile Team and they should be taken here.But they havent done that.So i hard coded that to see the working

File: src/services/WolfDen.API/appsettings.Development.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
         "SenderName": "WolfDen"
     },
     "ConnectionStrings": {
-        "DatabaseConnection":"Server=localhost,1431;Database=WolfDen;User Id=sa;Password=pass@123;TrustServerCertificate=true"
+        "DatabaseConnection":"Server=localhost,1431;Database=WolfDenUpdated;User Id=sa;Password=pass@123;TrustServerCertificate=true"
```
[the remark this answers is not in the document]
Comment (Asna-A) [reply src/services/WolfDen.API/appsettings.Development.json:14]: yes,when i tried to drop and migrate,it is only loading,no result was found.so i used a new new database

File: src/services/WolfDen.Application/Requests/Commands/LeaveManagement/LeaveTypes/AddLeaveType/AddLeaveTypeCommand.cs
[the remark this answers is not in the document]
Comment (Asna-A) [reply src/services/WolfDen.Application/Requests/Commands/LeaveManagement/LeaveTypes/AddLeaveType/AddLeaveTypeCommand.cs:1]: at that time,they havent pushed the validators which were used in this.So i cant build .So i deleted

**Review by sanalsunnyGW — APPROVED** [review]

**Pull request merged**